exporter/collector/googlemanagedprometheus/extra_metrics_test.go

```diff
@@ -66,6 +66,7 @@ func TestAddExtraMetrics(t *testing.T) {
 				metric := sm.Metrics().AppendEmpty()
 				metric.SetName("target_info")
 				metric.SetEmptyGauge().DataPoints().AppendEmpty().SetIntValue(1)
+				metric.Gauge().DataPoints().At(0).Attributes().PutStr(locationLabel, "us-east")
```

Comment:
@dashpole when we only drop service.name/etc, do we want to drop GMP mr labels too? Like here, `location` will get added to the monitored resource *and* as a metric label... that won't work right?

Reply:
Yeah, that sounds right.  Drop it if it is one of the reserved labels ("location", "cluster", "namespace", etc), or if it is a service.name/service.namespace/service.instance.id label